mobileapp/www/js/index.js

```diff
@@ -222,21 +234,37 @@ var app = {
       app.login();
     });
 
+    $("#use-touchid").click(function (e) {
+      if (window.localStorage.touchidLogin == 0) {
```

Comment:
Unless you set localStorage.touchidLogin to a default of 0 on start, just do: 
if (!window.localStorage.touchidLogin)